### ISSUE TITLE
variable `free_disk_space_mb` doesn't appear

### ISSUE THREAD

**Issue by oskapt** [bitfocus/companion-module-obs-studio#260]
**Describe the bug**

This is probably a PEBKAC issue, but I'm not able to find a solution in my searches.

I want to use the `free_disk_space_mb` variable in a feedback, but the only variable that's available is `free_disk_space`, which is labeled in `GB` and can't be used for value comparison.

This variable appeared in v3.4.0, and on Feb 9 Companion merged v3.4.3 into `main`. A new release was cut for Companion on Feb 26 (3.2.2), so I would expect that this variable would be present. It isn't.

Perhaps I'm looking in the wrong place? I considered that obs module version 3.4.3 != companion version 3.2.2, but it doesn't look like version numbers are synced between modules and the main companion app itself. I'm not sure where else to look.

**To Reproduce**

1. Create a button
2. Go the variables tab
3. Search for `disk` and only one variable appears

**Version Information:**

- Operating System: MacOS 13.5
- OBS Version: 30.1.0
- Companion Version: 3.2.2

![image](https://github.com/bitfocus/companion-module-obs-studio/assets/171292/d407bdd0-29cd-4b85-bd33-07bd98ca6a33)

**Comment by thedist** [comment]
I think you're confusing the branches and releases.

Companions `main` branch is the beta branch, so when modules get updated in that branch it'll only be reflected in new beta builds.

When Companion has a new release, because of SemVer when there's a patch release, eg going from Companion v3.2.1 to v3.2.2, it would only include patch changes. If modules had new features or functionality that would justify a minor version bump, they'd be included in the next Companion minor release, eg v3.3.0 (which is currently in beta testing).

tl;dr, if you want to use new features in a module, use Companions beta builds, or wait until there's a major or minor release of Companion rather than a patch release.
